static/js/constants.js

```diff
@@ -540,11 +550,24 @@ export const PROGRAM_ENROLLMENTS = [
   },
 ];
 
+export const FINANCIAL_AID_PARTIAL_RESPONSE = {
+  application_status: null,
```

Comment:
if the user hasn't started the application process, `has_user_applied: false` already indicates that and i don't think we need a specific status value to reinforce that